docs/source/quickstart.rst

```diff
@@ -141,8 +141,11 @@ simple demonstration of using Runway to deploy a Serverless Framework backend wi
     Remove-Item 35a66d144d8def340278cd55080d5c745714aca4.zip -Force
     Rename-Item angular-realworld-example-app-35a66d144d8def340278cd55080d5c745714aca4 frontend
     cd frontend
-    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"if test \`"`$(pipenv run runway whichenv)\`" = \`"prod\`" ; then ng build --prod --base-href .\/ && cp CNAME dist\/CNAME; else ng build --base-href .\/ && cp CNAME dist\/CNAME; fi`"," | Out-File .\package.json -Force -Encoding UTF8
+    (gc .\package.json -raw).Replace("`"rxjs`": `"^6.2.1`"", "`"rxjs`": `"~6.3.3`"") | sc .\package.json
+    Invoke-WebRequest https://raw.githubusercontent.com/onicagroup/runway/master/quickstarts/conduit/build.js -OutFile scripts/build.js
+    $(gc .\package.json) -replace "^\s*`"build`":\s.*$", "    `"build`": `"node scripts/build`"," | Out-File .\package.json -Force -Encoding UTF8
```

Comment:
This looks good to me -- can you change the posix example to use the build script too so they stay in sync?

Reply:
yeah, we can do that